record/record.app.js

```diff
@@ -50,6 +52,8 @@
                     $rootScope.relatedReferences = $rootScope.reference.related;
                     // There should only ever be one entity related to this reference
                     return $rootScope.reference.read(1);
+                }, function error(exception) {
+                    return $q.reject(exception);
```

Comment:
Ya, it is caught in the next then error callback, but I am changing it to throw the exception, so it is consistent with the other error callbacks.